src/sentry/integrations/vercel/integration.py

```diff
@@ -170,13 +192,30 @@ def build_integration(self, state):
                 "installation_id": data["installation_id"],
                 "installation_type": installation_type,
                 "webhook_id": webhook["id"],
+                "configurations": configurations,
+            },
+            "post_install_data": {
+                "user_id": state["user_id"],
+                # new configuration data
+                "configuration_id": data["installation_id"],
+                "access_token": access_token,
+                "webhook_id": webhook["id"],
             },
-            "post_install_data": {"user_id": state["user_id"]},
         }
 
         return integration
 
     def post_install(self, integration, organization, extra=None):
+        # add new configuration information to metadata
+        configurations = integration.metadata.get("configurations") or {}
+        configurations[extra["configuration_id"]] = {
+            "access_token": extra["access_token"],
+            "webhook_id": extra["webhook_id"],
+            "organization_id": organization.id,
+        }
+        integration.metadata["configurations"] = configurations
+        integration.save()
```

Comment:
I haven't seen this syntax before. Is `integration` a JSON object?

Reply:
the `integration` is the model instance, so im calling `save()` to make sure the integration metadata gets updated.